tsconfig.json

```diff
@@ -1,45 +1,10 @@
 {
+  "extends": "@backstage/cli/config/tsconfig.json",
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
The file I'm extending: https://github.com/backstage/backstage/blob/master/packages/cli/config/tsconfig.json